### DISCUSSION TITLE
Customize the `&` in arbitrary variant

### DISCUSSION THREAD

**Discussion by sntran** [tailwindlabs/tailwindcss#8779] [Ideas]
At work, I use XML to build HTML page, and it does not like the `&`, requiring it to be changed to `&amp;`. However, when Tailwind sees `&amp;`, it does not like that either, due to the semicolon.

I would like to request an config option to change the `&` to something else that won't be conflicting.

**Comment by brianseidman** [comment]
Same problem. I'd add that 
```
<div class="[&amp;_p]:text-red-500">
  <p>This text is red.</p>
</div>
```

works in the [Tailwind playground](https://play.tailwindcss.com/dWn8ncTz0c), but doesn't work when compiling a Tailwind site with the CLI.
